modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -218,7 +220,7 @@ private ThemeDisplay _getThemeDisplay() {
 
 	private Object _getValue(
 			ObjectField objectField, Map<String, Serializable> values)
-		throws PortalException {
+		throws ParseException, PortalException {
```

Comment:
@danvb112 I was considering DateUtil would be treating this exception... 

We should follow this example for handling this exception: https://github.com/liferay-objects/liferay-portal/blob/master/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java#L348-L360

Reply:
@danvb112 nevermind. Just throw `Exception` instead of these two exceptions.